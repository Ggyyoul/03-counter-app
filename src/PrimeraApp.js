const PrimeraApp = () => {
  return <div>Hola mundo</div>;
};

export default PrimeraApp;
