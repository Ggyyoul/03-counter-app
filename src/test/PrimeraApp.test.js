// import { render } from "@testing-library/react"

import {shallow} from 'enzyme'
import PrimeraApp from "../PrimeraApp"

describe('test in <PrimeraApp />', () => {

  // test('should look message "hola soy juan"', () => {
  //   const saludo = 'hola soy juan'

  //   const wrapper = render(<PrimeraApp saludo ={saludo}/>)

  //   expect(wrapper.getByText(saludo)).toBeInTheDocument();
  // })
  
  test('should look the component <PrimeraApp /> correctly', () => {

    const saludo = 'Hola soy Juan'
    const wrapper = shallow(<PrimeraApp  saludo={saludo}/>)
    expect(wrapper).toMatchSnapshot();
  })
  
})
