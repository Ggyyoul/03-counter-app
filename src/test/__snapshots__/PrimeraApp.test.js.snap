// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test in <PrimeraApp /> should look the component <PrimeraApp /> correctly 1`] = `
<Fragment>
  <h1>
    Hola soy Juan
  </h1>
  <p>
    Mi primer componente
  </p>
</Fragment>
`;
